### ISSUE TITLE
Variables in for-loops don't support '.' character

### ISSUE THREAD

**Issue by jgorgolewski** [Netcentric/accesscontroltool#330]
You cannot use '.' (dot) characters in for loops. It's similar issue to one reported in #14 

`private final Pattern forLoopPattern = Pattern.compile("for (\\w+) in \\[([,/\\s\\w\\-]+)\\]", Pattern.CASE_INSENSITIVE);`

shoud be probably extended to:

`private final Pattern forLoopPattern = Pattern.compile("for (\\w+) in \\[([,/\\s\\w\\-.]+)\\]", Pattern.CASE_INSENSITIVE);`

**Comment by ghenzler** [comment]
@jgorgolewski Please provide an example on how exactly you would like to use this feature. Is the `.` part of a literal? Or would you like to use variables in there (then  `${}` would also be needed)

**Comment by jgorgolewski** [comment]
@ghenzler Sure, right now literals are needed, for example:
```
- FOR domain IN [ example.com, example.co.uk ]:
  - example-mapping-manager:
    - path:  /etc/map/https/${domain}
      permission: allow
      actions: read,write
```

Ideally, FOR loop should support any characters in literals (which would probably require significant changes to allow escaping of ',', ' ' and ']'). Some intermediate step would be to allow all valid oak path characters: https://jackrabbit.apache.org/oak/docs/constraints.html

Support for variables could also be a nice feature but I do not have real use case for it right now.

**Comment by ghenzler** [comment]
@jgorgolewski Thanks for the example. Spaces and most character should already work because of `\w\s` in regex, I added the `\.` to be allowed and a unit test. I'll leave support for `, ` and `[`, `]` out for now to keep it simple

**Comment by ghenzler** [comment]
Fixed in v2.2.0
